### PR TITLE
Set more modules as upgrade-only

### PR DESCRIPTION
**Pull request by emteknetnz**

### CODE REVIEW

File: .cow.json [hunk 1]
```diff
@@ -19,7 +19,13 @@
...
-    "undefinedoffset/sortablegridfield",
-    "silverstripe/subsites"
+    "silverstripe/lumberjack",
+    "silverstripe/subsites",
+    "symbiote/silverstripe-gridfieldextensions",
+    "symbiote/silverstripe-multivaluefield",
+    "tractorcow/classproxy",
+    "tractorcow/silverstripe-fluent",
+    "tractorcow/silverstripe-proxy-db",
+    "undefinedoffset/sortablegridfield"
```
Comment (Cheddam) [inline .cow.json]: A lot of these modules aren't actually included in the list of modules Cow attempts to release anyway, so we can shorten this significantly.

```suggestion
    "silverstripe/subsites",
    "tractorcow/silverstripe-fluent",
```

**Comment by emteknetnz** [comment]
Travis failure is, strange?

image

@Cheddam any idea?  Related to composer 2 possibly (though haven't seen this happen on any other modules)?

**Comment by Cheddam** [comment]
Looks like this failure emerged [2 weeks ago](https://travis-ci.com/github/silverstripe/cwp-recipe-kitchen-sink/jobs/429505827), and the most obvious difference between it and [the previous build](https://travis-ci.com/github/silverstripe/cwp-recipe-kitchen-sink/jobs/427892413) is Composer 2, so I'd put money on that being the cause.

**Review by Cheddam — CHANGES_REQUESTED** [review]
The subsites addition is working, but the `tractorcow/silverstripe-fluent` entry needs to go in [cwp/cwp-installer](https://github.com/silverstripe/cwp-installer/blob/2.7/.cow.json), as Cow only looks at one level of config for upgrade-only entries.

**Review by Cheddam — APPROVED** [review]
Looking good, will MOG (need to tidy up the commit message.)

**Comment by Cheddam** [comment]
The remaining failure is unrelated, merging.

**Pull request merged**